Hackney.Shared.HousingSearch/Gateways/Models/Processes/QueryableProcess.cs

```diff
@@ -1,22 +1,47 @@
+using Hackney.Shared.HousingSearch.Domain.Process;
 using Nest;
+using System;
 using System.Collections.Generic;
+using System.Linq;
 
 namespace Hackney.Shared.HousingSearch.Gateways.Models.Processes
 {
     public class QueryableProcess
     {
+        public Process Create()
```

Comment:
Shouldn't this be creating a QueryableProcess? Otherwise it'd be a factory method

Reply:
That's the same way it has been done for others as well example QueryableAsset

Reply:
So you can create a Process from a QueryableProcess.Create() method? I don't think that's right.
Shouldn't it be this:

```suggestion
        public QueryableProcess Create()
```

Reply:
I've updated the method